cfg_samples/kanata.kbd

```diff
@@ -656,7 +656,7 @@ If you need help, you are welcome to ask.
 ;; defined above), these allow you to perform actions when a combination of
 ;; input keys (a "chord") are pressed all at once (order does not matter).
 ;; Each combination/chord can perform a different action, allowing you to bind
-;; up to `2^n - 1` different actions to just `n` keys.
+;; up to `(n)(n+1)/2` different actions to just `n` keys.
```

Comment:
No, it's definitely exponential. Given `n` keys you can think of those representing a `n` bit number where each bit is set if the corresponding key was pressed (in fact, that's how `ChordKeys` is implemented, it's just a bit mask with exactly that behavior). With `n` bits, you can represent `2^n` different values. You cannot however press no keys at all (`0`) because you need to press at least one key to enter chording mode, hence `2^n - 1`.

The `binary` example chord group below has 4 keys (`1`, `2`, `4`, `8`) and 15 actions. `15 = 16 - 1 = 2^4 - 1 != (4)(4+1)/2 = 4*5/2 = 10`.

Reply:
Ah yes you're right, I'll correct it